src/m365/spo/commands/listitem/listitem-get.spec.ts

```diff
@@ -177,7 +177,7 @@ describe(commands.LISTITEM_GET, () => {
       webUrl: 'https://contoso.sharepoint.com/sites/project-x', 
       id: expectedId,
       output: "json",
-      fields: "ID,Modified"
+      properties: "ID,Modified"
```

Comment:
We should also update the name of the test so that it's referring to properties rather than fields